packages/cli/src/daemon-config.ts

```diff
@@ -262,6 +269,9 @@ export class DaemonConfig {
   @jsonMember(DaemonStateStoreConfig, { name: 'state-store' })
   stateStore: DaemonStateStoreConfig
 
+  @jsonMember(DaemonCeramicResolversConfig)
+  resolvers?: DaemonCeramicResolversConfig;
```

Comment:
```suggestion
  didResolvers?: DaemonCeramicResolversConfig;
```

This would be more clear imo

Reply:
okay

Reply:
done.